### DISCUSSION TITLE
Can we format constants more nicely?

### DISCUSSION THREAD

**Discussion by FrostyX** [gleam-lang/gleam#3976] [Ideas & suggestions]
I have this `let` variable:

```gleam
let map = [
  #("A", [Dot, Comma]),
  #("B", [Comma, Dot, Dot, Dot]),
  #("C", [Comma, Dot, Comma, Dot]),
  #("D", [Comma, Dot, Dot]),
  #("E", [Dot]),
  #("F", [Dot, Dot, Comma, Dot]),
  #("S", [Dot, Dot, Dot]),
]
```

The formatting is very readable. However, when I turn it into a constant, `gleam format` does this:

```gleam
const map = [
  #("A", [Dot, Comma]), #("B", [Comma, Dot, Dot, Dot]),
  #("C", [Comma, Dot, Comma, Dot]), #("D", [Comma, Dot, Dot]), #("E", [Dot]),
  #("F", [Dot, Dot, Comma, Dot]), #("S", [Dot, Dot, Dot]),
]
```

Can we change `gleam format` to do the same exact formatting for `const` as it does for `let`?

**Comment by lpil** [comment]
Ah! Yes, sorry. I would consider it a bug that they do not get formatted the same way.

**Reply by FrostyX** [reply]
Thank you for the response @lpil. I was a bit worried I am just misunderstanding something.
Do you want me to create an issue for this, or will you transfer this discussion to an issue?

**Reply by lpil** [reply]
I have made one! Sorry, I thought it would alert you https://github.com/gleam-lang/gleam/issues/4084
